docs/extensions/ResponseMetadata.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Response Metadata
+category: extensions
+order: 8
+---
+
+# Response Metadata
+
+This extension allows to limit the amount of control information in responses, adapted from [OData JSON Format Version 4.01 - 3.1](http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_ControllingtheAmountofControlInforma).
+
+The metadata can be applied to the $resultMetadata parameter of a request to influence how much control information will be included in the response.
+
+Sample request:
+
+```
+GET /v1.1/Things(1)?$resultMetadata=off
```

Comment:
The parameter also has influence on POST requests. Could you please also describe that here?

Reply:
I added more sample requests.